RingBuffer.fsx

```diff
@@ -7,15 +7,72 @@
 #load "RingBuffer.fs"
 open Hopac
 open Expecto
+open Hopac.Infixes
 
 let tests =
   testList "RingBuffer" [
     testCaseAsync "enqueue" (job {
-      let! rb = RingBuffer.create 2u
+      let! rb = RingBuffer.create 4us
+
+      let mutable result = []
+      
+      let takePerSecond = Job.delay <| fun () ->
+        timeOutMillis 100 ^=>. job {
```

Comment:
Should be << 100 ms, because on L35, you wait the sum of 10x100; so `result` is too racy (should complete test on same instant as this loop completes). Why not just loop through and get 7 items from the buffer?